cosmos/operators/local.py

```diff
@@ -538,13 +538,85 @@ class DbtDocsLocalOperator(DbtLocalBaseOperator):
 
     ui_color = "#8194E0"
 
-    required_files = ["index.html", "manifest.json", "graph.gpickle", "catalog.json"]
+    required_files = ["index.html", "manifest.json", "graph.gpickle", "catalog.json", "run_results.json"]
 
     def __init__(self, **kwargs: Any) -> None:
         super().__init__(**kwargs)
         self.base_cmd = ["docs", "generate"]
 
 
+class DbtFreshnessLocalOperator(DbtLocalBaseOperator):
+    """
+    Executes `dbt source freshness` command.
+    Use the `callback parameter to specify a callback function to run after the command completes.
+    """
+
+    ui_color = "#8194E0"
+
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        self.base_cmd = ["source", "freshness"]
+
+
+class DbtFreshnessS3LocalOperator(DbtFreshnessLocalOperator):
```

Comment:
Similar here, I wonder if `DbtSourceFreshnessS3LocalOperator` would be more representative.